### PR TITLE
Add PR preview

### LINKED ISSUE #166: Add a workflow to preview PRs
See https://github.com/gmt-china/GMT_docs/blob/master/.github/workflows/preview-pr.yml for inspirations.

### PR DESCRIPTION
**Pull request by core-man**
Fixes #166.

### CODE REVIEW

**Comment by core-man** [comment]
See https://github.com/gmt-china/gmt-china.org/runs/2705187117?check_suite_focus=true:
> Error: Error building site: "/home/runner/work/gmt-china.org/gmt-china.org/content/blog/2016-04-06-gmt-china-community-founded/index.md:53:1": failed to extract shortcode: template for shortcode "alert" not found

File: .github/workflows/preview-pr.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.9.0
+
+      - name: Checkout
+        uses: actions/checkout@v2.3.4
```
Comment (seisman) [inline .github/workflows/preview-pr.yml:17]: You need to add submodule option herr

Comment (core-man) [inline .github/workflows/preview-pr.yml:17]:
```suggestion
        uses: actions/checkout@v2.3.4
        with: 
          submodules: 'true'
```

Comment (core-man) [reply .github/workflows/preview-pr.yml:17]: Done in https://github.com/gmt-china/gmt-china.org/pull/179/commits/fd6d20d60df5d431860251e3497d52677d4598ed

**Review by seisman — APPROVED** [review]

**Pull request merged**